### ISSUE TITLE
Unable to embed Ruby code with string interpolation

### ISSUE THREAD

**Issue by leighhalliday** [ChristopherBiscardi/gatsby-mdx#245]
**Describe the bug**
When using with gatsby-remark-prismjs, Ruby code that contains string interpolation causes errors.

**To Reproduce**
Embed ruby code like this:

```ruby 
# config/routes.rb 
scope "(:locale)", locale: /#{MySite::ROUTE_LOCALES.keys.join("|")}/ do 
  root :to => 'home#index' 
  resources :pages 
  # etc... 
end 
```

**Expected behavior**
To render without error.

**Screenshots**
<img width="1440" alt="screen shot 2019-01-12 at 2 45 49 pm" src="https://user-images.githubusercontent.com/603921/51077771-d1028280-1678-11e9-9b42-208ff97e5055.png">

**Additional context**
None.

**Comment by ChristopherBiscardi** [comment]
yeah, since `gatsby-remark-prismjs` inserts [chunks of html](https://github.com/gatsbyjs/gatsby/blob/a2f42ebc7d7510ceb70797ba25169960a20de9c4/packages/gatsby-remark-prismjs/src/index.js#L64-L73) into the AST, it doesn't get converted to JSX. The `{}` braces are being interpreted as template literal interpolations.

To get around this you can help upgrade gatsby-remark-prismjs to insert HAST instead, or you can [use components](https://www.christopherbiscardi.com/post/codeblocks,-mdx,-and-mdx-utils) and an `MDXProvider` instead.

**Comment by ChristopherBiscardi** [comment]
this should be fixed in `@mdx-js/mdx@0.19`
